src/parse.js

```diff
@@ -130,6 +135,13 @@ Parser.prototype.parseF2Dot14 = function() {
     return v;
 };
 
+
+Parser.prototype.parseUInt24 = function() {
+    const v = getUInt24(this.data, this.offset + this.relativeOffset);
```

Comment:
There's another PR that implements this function, I can't remember which one but that PR will need to be refactored if this is merged.

Reply:
ok, whatever gets merged first, the other one will have to remove it